### DISCUSSION TITLE
Does "PotentialEP" include soil evaporation?

### DISCUSSION THREAD

**Discussion by BrianCollinss** [APSIMInitiative/ApsimX#8869] [Q&A] [answered]
Does `PotentialEP` include soil evaporation? If not, why is it called `potential evapotranspiration`?

**Comment by hol353** [comment]
PotentialEP does not include soil evaporation. It is calculated by MicroClimate and passed to the leaf organ in the plant. It is the evaporative demand by a leaf on a given day i.e the maximum amount it can transpire. There may not be sufficient water in the system for the plant to transpire this much though - hence 'potential'.

**Reply by BrianCollinss** [reply]
Thanks Dean. I think it'd be helpful to correct the name of this variable (evaporation instead of evapotranspiration).

**Comment by hol353** [comment]
No, we can't change the name of variable. Most modellers know it as PotentialEP. Changing it will confuse a lot of modlelers.

**Reply by BrianCollinss** [reply]
I meant the definition.

**Reply by BrianCollinss** [reply]
![image](https://github.com/APSIMInitiative/ApsimX/assets/39573622/dda27dd6-d37b-49af-8d6a-3511a2dee37e)

**Reply by hol353** [reply]
The summary description seems ok to me. What would you change it to?

**Reply by HamishBrownPFR** [reply]
I think the summary could more correctly say "potential transpiration" rather than "potential evapotranspiration"
Microclimate returns the calculated Penman-Monteith potential transpiration under the name PotentialEP where I guess the P means Plant.  This value is not really appropriate to use as a potential for soil evaporation because it accounts for canopy height and leaf area index in its formulation of aerodynamic conductance whereas aerodynamic conductance from the soil is not dependent on those factors.  I think that is why soil still calculates its own potential evaporation.

**Comment by hut104** [comment, marked as answer]
Ep is potential for plant(s)
Eo for a potential atmospheric demand (what ever that might mean)
Eos is potential from the soil
Es is actual from the soil
________________________________

**Comment by BrianCollinss** [comment]
If we wound want to use FAO-approved terms, is Eo "reference evapotranspiration"?

**Comment by hut104** [comment]
Eo is not a reference evaporation rate within APSIM.
The reference in FAO56 is for a grass of LAI = 2, 20cm height, etc....
There is no reference crop in the concept of Eo in APSIM.

________________________________
